substrate/frame/revive/src/storage.rs

```diff
@@ -566,6 +567,25 @@ impl<T: Config> DeletionQueueManager<T> {
 	}
 }
 
+/// Debugging settings that can be configured when DebugEnabled config is true.
+#[derive(
+	DefaultNoBound,
+	Encode,
+	Decode,
+	CloneNoBound,
+	PartialEq,
+	Eq,
+	RuntimeDebug,
+	TypeInfo,
+	MaxEncodedLen,
+	Serialize,
+	Deserialize,
+)]
```

Comment:
```suggestion
#[derive(
	Default,
	Encode,
	Decode,
	Clone,
	PartialEq,
	Eq,
	Debug,
	TypeInfo,
	MaxEncodedLen
)]
```
you don't need the NoBound here since there are no generic, I don't think we need Serialize / Deserialize as well, not sure about the other